test/Spec.hs

```diff
@@ -315,6 +314,12 @@ validShowURLSpec =
                 let typedURL    = showURL $ TicketCommentsURL ticketId
                     untypedURL  = "/tickets/" <> show (getTicketId ticketId) <> "/comments.json"
                 in  typedURL == untypedURL
+        it "returns valid ExportDataByTimestamp" $ do
```

Comment:
This test is failing

Reply:
Fixed. Speaking of failing tests, did you add these:
-      processes ticket, with comments FAILED [1]
-      processes ticket, with no attachments FAILED [2]
-      processes ticket, with attachments FAILED [3]

Reply:
Yes I did, and it's strange because those tests are passing in the develop branch.